gemfiles/rails_5.gemfile

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Shouldn't this file be deleted if you are no longer doing Rails 5?

Reply:
I though `bundle exec appraisal install` would get rid of it? I guess not.